### DISCUSSION TITLE
How to disable the `@media (prefers-reduced-motion: reduce)` css

### DISCUSSION THREAD

**Discussion by justforuse** [vuejs/vitepress#3846] [General]
The critical code is here: https://github.com/vuejs/vitepress/commit/fc5092fb651487e69737fff04d3979f00c67dcc6

I'm working on Mac, and prefer to turn on “Reduce motion.” because I dont like its screen animation. But it close all the vitepress website's animation also, due to these CSS code. like loading or other animation.

Is there anyway to not include these code in Vitepress? Or how to disable it in themeConfig

**Comment by mistic100** [comment]
Here what I had to do to have a working spinner even in reduce motion mode.

```css
@layer __vitepress_base {
  .spinner:before {
    animation: .... !important;
  }
}
